src/ReactiveUI/Platforms/netstandard2.0/PlatformRegistrations.cs

```diff
@@ -3,14 +3,16 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Reactive.Concurrency;
 
 namespace ReactiveUI
 {
     public class PlatformRegistrations : IWantsToRegisterStuff
     {
         public void Register(Action<Func<object>, Type> registerFunction)
         {
-            throw new Exception("You are referencing the Portable version of ReactiveUI in an App. Reference the platform-specific version.");
```

Comment:
This is a big change that I'm not comfortable with. We used to throw if the end user didn't reference the right packages as part of bait and switch now we don't. Now it'll be harder for the end user and us to troubleshoot registrations as previously we had certainty that things would explode. Thoughts?